packages.bzl

```diff
@@ -2,7 +2,7 @@
 # all in-sync. This map is passed to each ng_package rule to stamp out the appropriate
 # version for the placeholders.
 ANGULAR_PACKAGE_VERSION = "^11.0.0 || ^12.0.0-0"
-MDC_PACKAGE_VERSION = "^9.0.0-canary.419e03572.0"
+MDC_PACKAGE_VERSION = "^11.0.0-canary.3201cae47.0"
```

Comment:
Should the version in the package.json be changed too? It's what we use for the CI.

Reply:
I believe that it matches this version due to https://github.com/angular/components/pull/21968.